src/main/java/com/sivalabs/ft/features/domain/FeatureDependencyRepository.java

```diff
@@ -0,0 +1,36 @@
+package com.sivalabs.ft.features.domain;
+
+import com.sivalabs.ft.features.domain.entities.FeatureDependency;
+import com.sivalabs.ft.features.domain.models.DependencyType;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.ListCrudRepository;
+
+interface FeatureDependencyRepository extends ListCrudRepository<FeatureDependency, Long> {
+
+    List<FeatureDependency> findByFeature_Code(String featureCode);
+
+    List<FeatureDependency> findByDependsOnFeature_Code(String dependsOnFeatureCode);
```

Comment:
findByFeature_Code is specified in description.

But all other methods are not, and used in tests.

This should be fixed.